### ISSUE TITLE
Running tests

### ISSUE THREAD

**Issue by joseluistorres** [nathanvda/cocoon#230]
Hi @nathanvda I forked your repo and run bundle install, then I run bundle exec spec but I get a failing test, am I missing a step?

```
bundle exec rake spec
/Users/joseluistorres/.rvm/rubies/ruby-2.1.1/bin/ruby -I/Users/joseluistorres/.rvm/gems/ruby-2.1.1/gems/rspec-core-3.0.4/lib:/Users/joseluistorres/.rvm/gems/ruby-2.1.1/gems/rspec-support-3.0.4/lib -S /Users/joseluistorres/.rvm/gems/ruby-2.1.1/gems/rspec-core-3.0.4/exe/rspec ./spec/cocoon_spec.rb ./spec/generators/install_generator_spec.rb ./spec/integration/navigation_spec.rb
Coverage report generated for RSpec to /Users/joseluistorres/gem-sources/cocoon/coverage. 77 / 193 LOC (39.9%) covered.

/Users/joseluistorres/.rvm/gems/ruby-2.1.1/gems/activesupport-4.1.5/lib/active_support/dependencies.rb:247:in
 `require': cannot load such file -- rspec/rails/mocks (LoadError)
```

**Comment by simi** [comment]
@nathanvda I think this is solved now.

**Comment by nathanvda** [comment]
Yep. Fixed in #252.
